scripts/release

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+if [[ ! -d releases ]]; then
+  echo -n "---> Which git repo do you want to use for releases? "
+  read GIT_REPO
+  git clone $GIT_REPO releases
+fi
+
+echo -n "---> Name for release tag? "
+read RELEASE_VERSION
+npm run build
```

Comment:
Wouldn't you want your `RELEASE_VERSION` to be in the package.json / output files?